### PR TITLE
[stable/sonarqube] Allow use of custom cacerts

### PR DESCRIPTION
**Pull request by papanito**
#### What this PR does / why we need it:

In environments with air-gapped setup, especially with internal tooling (repos) and self-signed certificates it is required to provide an adequate cacerts which overrides the default one:

#### Which issue this PR fixes
none

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by rjkernick** [comment]
Please fix the DCO: https://github.com/helm/charts/pull/14413/checks?check_run_id=140683332

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 1.0.4
+version: 1.0.5
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please bump the minor version as this adds a new feature

Comment (papanito) [reply stable/sonarqube/Chart.yaml]: Ok will do so.

**Comment by papanito** [comment]
Signoff is already fixed.

**Comment by rjkernick** [comment]
> Signoff is already fixed.

Not according to the bottom checks

**Comment by papanito** [comment]
@rjkernick thanks for the review, now all my changes seem fine.

File: stable/sonarqube/Chart.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 1.0.5
+version: 1.1.0
 appVersion: 7.7
 keywords:
   - coverage
   - security
-  - code
+  - cod
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please fix the typo

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
lgtm

**Comment by papanito** [comment]
Thanks @rjkernick

**Pull request merged**